neo/Consensus/ConsensusContext.cs

```diff
@@ -138,7 +138,7 @@ public bool Load()
             }
         }
 
-        public ConsensusPayload MakeChangeView(byte newViewNumber)
+        public ConsensusPayload MakeChangeView()
         {
             return ChangeViewPayloads[MyIndex] = MakeSignedPayload(new ChangeView
             {
```

Comment:
Why we don't send the expectedView in this payload?

Reply:
The expectedView is going inside the ConsensusMessage...
ChangeView inherits from ConsensusPayload, and ConsensusPayload has a ConsensusMessage.ViewNumber, we use that.